ML_Project.py

```diff
@@ -123,9 +124,32 @@
 # Removed any rows missing data. Should only remove 2 rows missing embarked data.
 combined_df = combined_df.dropna()
 
+## Making embarkation data usable
+for dataset in combine:
+    dataset['Embarked'] = dataset['Embarked'].map( {'S': 0, 'C': 1, 'Q': 2} ).astype(int)
```

Comment:
This is giving me a:
`ValueError: Cannot convert non-finite values (NA or inf) to integer`